config/environments/development.rb

```diff
@@ -14,7 +14,7 @@
   config.public_file_server.enabled        = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
I was testing out the password reset email, it wasn't working in dev so I was doing some random troubleshooting. I'll leave this for future PR.